usecases/recording.cameras.opencv.3/README.md

```diff
@@ -0,0 +1,72 @@
+This folder provides the instructions for testing proxy-camera, which can be executed from a docker-compose file to start all micro-services to record video streams from three OpenCV cameras with lossless H264 compression. It is assumed that git, Docker, and Docker Compose are installed and all the three cameras are properly connected. To install Docker, follow the tutorial: https://docs.docker.com/engine/installation/linux/ubuntulinux/
+
+### Prepare OpenDaVINCI Docker image
+
+Clone the OpenDaVINCI source tree and get the latest update:
+
+    $ git clone https://github.com/se-research/OpenDaVINCI
+    
+    $ git pull
+    
+Go to the docker folder and build OpenDaVINCI Docker image:
+
+    $ make
+    
+### Prepare proxy-camera
+
+proxy-camera is included in the feature.camera branch of the opendlv.core repository (https://github.com/chalmers-revere/opendlv.core). Clone the opendlv.core source and switch to the feature.camera branch:
+
+    $ git clone https://github.com/chalmers-revere/opendlv.core
+    
+    $ git checkout feature.camera
+    
+    $ git pull
+    
+Go to opendlv.core/docker, build and create the Docker image seresearch/opendlv-core-on-opendavinci-ubuntu-16.04-complete:latest:
+
+    $ make buildComplete
+    
+    $ make createDockerImage
+    
+### Test proxy-camera with Docker Compose
```

Comment:
Why can't we simply let the configuration file reside next to the docker-compose file and the recordings being created in /opt/recordings?